packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -103,10 +103,10 @@ contract L2OutputOracle is OwnableUpgradeable, Semver {
     mapping(uint256 => OutputProposal) internal l2Outputs;
 
     /**
-     * @notice Reverts if called by any account other than the sequencer.
```

Comment:
Perhaps we want to rename the `l2Outputs` mapping as well?

Reply:
I did play around with that, but didn't find anything I liked more. I considered: 
- `outputProposals`: but they are only proposals until they are finalized then they are outputs
- `outputs`: I do find the `l2` prefix a bit clunky and repetitive in this contract. However `output` is a vague term, and `l2` makes it more readable especially for external devs.